addons/skin.estuary/xml/Variables.xml

```diff
@@ -831,31 +831,94 @@
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dtshd_hra)">DTS-HD HRA</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dtsma)">DTS-HD MA</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dtshd_ma_x_imax)">DTS-HD MA / DTS:X IMAX</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dtshd_ma_x)">DTS-HD MA / DTS:X</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dtshd_ma)">DTS-HD MA</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dca)">DTS</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dts)">DTS</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),eac3_ddp_atmos)">Dolby Digital+ Atmos</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),eac3)">Dolby Digital+</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),dolbydigital)">Dolby Digital</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),ac3)">Dolby Digital</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),truehd_atmos)">Dolby TrueHD Atmos</value>
+		<value condition="Window.IsActive(dialogselectaudio) + String.Contains(ListItem.Property(stream.codec),truehd)">Dolby TrueHD</value>
+		<value>$INFO[ListItem.Property(stream.codec)]</value>
+	</variable>
 	<variable name="MediaInfoListLabelVar">
 		<value condition="Window.IsVisible(selectvideoversion)">$INFO[ListItem.VideoVersionName]</value>
 		<value>$INFO[ListItem.Label]</value>
 	</variable>
 	<variable name="MediaInfoListLabel2Var">
-		<value condition="ListItem.IsStereoscopic">$INFO[ListItem.Duration,$LOCALIZE[180]: ][CR]3D | $VAR[VideoCodecVar]$INFO[ListItem.VideoResolution, | ]$VAR[VideoResolutionTypeVar, ]$VAR[VideoHDRTypeVar, | ]$INFO[ListItem.VideoAspect, | ,:1]$VAR[AudioListCodecVar, | ]$VAR[AudioChannelsVar, ]</value>
-		<value>$INFO[ListItem.Duration,$LOCALIZE[180]: ][CR]$VAR[VideoCodecVar]$INFO[ListItem.VideoResolution, | ]$VAR[VideoResolutionTypeVar, ]$VAR[VideoHDRTypeVar, | ]$INFO[ListItem.VideoAspect, | ,:1]$VAR[AudioCodecVar, | ]$VAR[AudioChannelsVar, ]</value>
+		<value condition="ListItem.IsStereoscopic">$INFO[ListItem.Duration,$LOCALIZE[180]: ][CR]3D | $VAR[VideoCodecVar]$INFO[ListItem.VideoResolution, | ]$VAR[VideoResolutionTypeVar, ]$VAR[VideoHDRTypeVar, | ]$INFO[ListItem.VideoAspect, | ,:1]$VAR[AudioCodecVar,$COMMA ]$VAR[ObjectAudioTypeVar, / ]$VAR[AudioChannelsVar, ]</value>
```

Comment:
This is just personal choice but I did prefer the vertical bar separator.

Reply:
I went with comma's so it's consistent with have stream details are presented in the stream selection dialogs.

Reply:
But I could perhaps try other way and use vertical bar separator in all places, but that would require further width adjustments, as if I remember correctly we went with comma's in stream selection dialogs as it more space efficient.

Reply:
Decided to go with vertical separators everywhere, screenshots updated to show this.